cli/internal/cmd/auth/logout.go

```diff
@@ -0,0 +1,79 @@
+package auth
+
+import (
+	"errors"
+	"fmt"
+
+	"github.com/fatih/color"
+	"github.com/vercel/turborepo/cli/internal/config"
+	"github.com/vercel/turborepo/cli/internal/ui"
+	"github.com/vercel/turborepo/cli/internal/util"
+
+	"github.com/mitchellh/cli"
+	"github.com/spf13/cobra"
+)
+
+// LogoutCommand is the structure for the logout command
+type LogoutCommand struct {
+	Config *config.Config
+	UI     *cli.ColoredUi
+}
+
+// Synopsis of the logout command
+func (c *LogoutCommand) Synopsis() string {
+	return LogoutCmd(c).Short
+}
+
+// Help returns information about the logout command
+func (c *LogoutCommand) Help() string {
+	return util.HelpForCobraCmd(LogoutCmd(c))
+}
+
+// Run setups the command and runs it
+func (c *LogoutCommand) Run(args []string) int {
+	cmd := LogoutCmd(c)
+
+	cmd.SilenceErrors = true
```

Comment:
I think we want to set `SilenceUsage` as well. Alternatively, I believe you can set them both inline when creating the command.